radial_hit.m

```diff
@@ -1,12 +1,6 @@
-function[tMaxR, tStepR, transition_flag]=radial_hit(ray_origin, ray_direction, current_radial_voxel, ...
-        circle_center, circle_max_radius, delta_radius, t, ray_unit_vector, ray_circle_vector, v, prev_transition_flag, verbose)
+function[tMaxR, tStepR, transitionFlag]=radial_hit(ray_origin, ray_direction, current_radial_voxel, ...
```

Comment:
I switched it from camel case to snake case to remain consistent with the rest of our code, and since yt is using snake case. Is there a reason why its switched back?